automatic/datagrip/tools/ChocolateyUnInstall.ps1

```diff
@@ -0,0 +1,16 @@
+﻿$extractionPath = (${env:ProgramFiles(x86)}, ${env:ProgramFiles} -ne $null)[0]
```

Comment:
Don't use the program files environment variables, there is no guarantee the user have the software installed there.
Better to use the registry keys to detect the path (look at the comment below)